manual/import-tool/src/test/java/org/neo4j/tooling/ImportToolDocIT.java

```diff
@@ -696,10 +693,9 @@ private void printCommandToFile( String[] arguments, String dir, String fileName
             }
         }
         String documentationArgs = StringUtils.join( cleanedArguments, " " );
-        documentationArgs =
-                documentationArgs.replace( dir + File.separator, "" ).replace(
-                        directory.absolutePath().getAbsolutePath(), "path_to_target_directory" );
-        String docsCommand = "neo4j-import " + documentationArgs;
+        documentationArgs = documentationArgs.replace( dir + File.separator, "" )
+                .replace( directory.absolutePath().getAbsolutePath(), "path_to_target_directory" );
+        String docsCommand = "neo4j-admin import --mode=csv" + documentationArgs;
```

Comment:
Aren't you specifying `--mode=csv` in the individual tests? Maybe you can remove it there.